invenio_stats/utils.py

```diff
@@ -35,8 +35,14 @@ def get_anonymization_salt(ts):
     return salt
 
 
-def get_bucket_size(client, index, agg_field):
+def get_bucket_size(client, index, agg_field, start_date=None, end_date=None):
     """Function to help us define the size for our search query."""
+    time_range = {}
+    if start_date is not None:
+        time_range['gte'] = start_date.replace(microsecond=0).isoformat()
```

Comment:
Since `start_date` and `end_date` are strings here, this will probably fail. Since we don't have Python 3 type annotations, adding a docstring for this utility is a must to promote correct usage.